content/docs/command-reference/exp/index.md

```diff
@@ -26,19 +26,19 @@ usage: dvc exp [-h] [-q | -v]
 
 positional arguments:
   COMMAND
+    init                Quickly setup any project to use DVC Experiments.
+    run                 Reproduce complete or partial experiment pipelines.
     show                Print experiments.
-    apply               Apply the changes from an experiment to your
-                        workspace.
     diff                Show changes between experiments in the DVC
                         repository.
-    run                 Reproduce complete or partial experiment pipelines.
-    gc                  Garbage collect unneeded experiments.
-    branch              Promote an experiment to a Git branch.
     list                List local and remote experiments.
+    apply               Apply the changes from an experiment to your
+                        workspace.
+    branch              Promote an experiment to a Git branch.
+    remove              Remove local experiments.
+    gc                  Garbage collect unneeded experiments.
```

Comment:
Question @dberenbaum @skshetry : did we update the core repo with these texts? Thanks

Reply:
Wait... There are no changes to the text here ⌛